### PR TITLE
100% Coverage

### PR DESCRIPTION
**Pull request by 8W9aG**
- Fix the final remanants

### CODE REVIEW

**Comment by 8W9aG** [comment]
@spotify/objc-dev

**Comment by dflems** [comment]
:+1:

File: Tests/SPTPersistentCacheHeaderTests.m [hunk 1]
```diff
@@ -55,7 +55,7 @@ - (void)testValidateMisalignedHeader
 {
     #pragma clang diagnostic push
     #pragma clang diagnostic ignored "-Wcast-align"
-    int headerValidationResult = SPTPersistentCacheValidateHeader((SPTPersistentCacheRecordHeader *)"a");
+    int headerValidationResult = SPTPersistentCacheValidateHeader((SPTPersistentCacheRecordHeader *)3);
```
Comment (ChocoChipset) [inline Tests/SPTPersistentCacheHeaderTests.m:58]: Is this change intentional?
